packages/svgs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "@hopper-ui/svgs",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "The svgs package.",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gsoft-inc/wl-hopper.git",
+        "directory": "packages/svgs"
+    },
+    "publishConfig": {
+        "access": "public",
+        "provenance": true
+    },
+    "type": "module",
+    "files": [
+        "/dist",
+        "CHANGELOG.md",
+        "README.md"
+    ],
+    "scripts": {
+        "build": "node --loader ts-node/esm src/scripts/write-icons.ts",
+        "test": "jest --config jest.config.ts"
```

Comment:
just a FYI, but the test method in will be at the root of the mono repo only. so we'll remove this line one day. You can keep it to test for now